client/components/suggestions/index.jsx

```diff
@@ -113,27 +124,27 @@ const Suggestions = React.createClass( {
 				break;
 		}
 		return false;
-	},
+	}
 
-	onMouseDown( event ) {
+	onMouseDown = ( event ) => {
 		event.stopPropagation();
 		event.preventDefault();
 		//Additional empty space at the end adds fluidity to workflow
 		this.props.suggest( event.target.textContent + ' ' );
-	},
+	}
 
-	onMouseOver( event ) {
+	onMouseOver = ( event ) => {
 		this.setState( {
 			suggestionPosition: this.getPositionForSuggestion( event.target.textContent ),
 			currentSuggestion: event.target.textContent,
 		} );
-	},
+	}
 
-	removeEmptySuggestions( suggestions ) {
+	removeEmptySuggestions = ( suggestions ) => {
 		return pickBy( suggestions, negate( isEmpty ) );
-	},
+	}
 
-	narrowDown( input ) {
+	narrowDown = ( input, showAll = '' ) => {
```

Comment:
Can we have some jsdoc explaining what this function returns? It is not necessarily clear from the code that in  the `showAll` case, this func also does a bit of a sort.